website/src/routes/navigationItems.ts

```diff
@@ -1,34 +1,47 @@
+import type { ComponentType } from 'react';
+
 import { bottomNavigationItems } from './bottomNavigationItems.ts';
-import { extraSequenceRelatedTopNavigationItems, extraStaticTopNavigationItems } from './extraTopNavigationItems.js';
+import { extraStaticTopNavigationItems } from './extraTopNavigationItems.js';
 import { routes } from './routes.ts';
 import { getWebsiteConfig } from '../config.ts';
+import UploadIcon from '~icons/material-symbols/upload';
+import SearchIcon from '~icons/tabler/list-search';
 
 export const navigationItems = {
     top: topNavigationItems,
     bottom: bottomNavigationItems,
 };
 
-export type TopNavigationItems = {
+type NavigationIcon = ComponentType<{ className?: string }>;
+
+export type TopNavigationItem = {
     text: string;
     path: string;
-}[];
+    id?: string;
+    icon?: NavigationIcon;
+};
+
+export type TopNavigationItems = TopNavigationItem[];
+
+export function getSequenceRelatedItems(organism: string | undefined) {
+    if (organism === undefined) {
+        return [];
+    }
 
-function getSequenceRelatedItems(organism: string | undefined) {
     const browseItem = {
-        text: 'Browse',
-        path: organism !== undefined ? routes.searchPage(organism) : routes.organismSelectorPage('search'),
+        text: 'Browse data',
+        path: routes.searchPage(organism),
+        icon: SearchIcon,
     };
 
     if (!getWebsiteConfig().enableSubmissionNavigationItem) {
         return [browseItem];
     }
 
     const submitItem = {
-        text: 'Submit',
-        path:
-            organism !== undefined
-                ? routes.submissionPageWithoutGroup(organism)
-                : routes.organismSelectorPage('submission'),
+        text: 'Submit sequences',
```

Comment:
"Browse data" and "Submit sequences" are quite long, in particular for small screens. I've a small preference for sticking with the shorter "Browse" and "Submit"

<img width="664" height="202" alt="Image" src="https://github.com/user-attachments/assets/3c4fd866-a5e0-44b8-84ba-dd53b726f62e" />

Reply:
I agree that we should improve the mobile experience here (I also think we should change Crimean Co.. to CCHF - I have a draft PR to add tooltips to organisms to help with this). I also agree that these may not be ideal final tiles.

I'd prefer to tackle this downstream and not block on it atm. (Personally I think something a bit longer than "Browse" and "Submit" might be good - esp. given that there were concerns about visibility).